course/models.py

```diff
@@ -252,72 +301,97 @@ def __unicode__(self):
 # {{{ participation
 
 class ParticipationTag(models.Model):
-    course = models.ForeignKey(Course)
-
+    course = models.ForeignKey(Course,
+            verbose_name = 'Course ID')
     name = models.CharField(max_length=100, unique=True,
             help_text="Format is lower-case-with-hyphens. "
-            "Do not use spaces.")
+            "Do not use spaces.",
+            verbose_name = 'Name of ParticipationTag')
 
     def clean(self):
         import re
         name_valid_re = re.compile(r"^\w+$")
 
         if name_valid_re.match(self.name) is None:
+            # Translators: "Name" is the name of a ParticipationTag
             raise ValidationError({"name": "Name contains invalid characters."})
 
     def __unicode__(self):
         return "%s (%s)" % (self.name, self.course)
 
     class Meta:
+        verbose_name = "Participation tag"
+        verbose_name_plural = "Participation tags"
         unique_together = (("course", "name"),)
         ordering = ("course", "name")
 
 
 class Participation(models.Model):
-    user = models.ForeignKey(User)
-    course = models.ForeignKey(Course, related_name="participations")
+    user = models.ForeignKey(User,
+            verbose_name = 'User ID')
+    course = models.ForeignKey(Course, related_name="participations",
+            verbose_name = 'Course ID')
 
-    enroll_time = models.DateTimeField(default=now)
+    enroll_time = models.DateTimeField(default=now,
+            verbose_name = 'Enroll time')
     role = models.CharField(max_length=50,
             choices=PARTICIPATION_ROLE_CHOICES,
             help_text="Instructors may update course content. "
             "Teaching assistants may access and change grade data. "
             "Observers may access analytics. "
-            "Each role includes privileges from subsequent roles.")
+            "Each role includes privileges from subsequent roles.",
+            verbose_name = 'Participation role')
     status = models.CharField(max_length=50,
-            choices=PARTICIPATION_STATUS_CHOICES)
+            choices=PARTICIPATION_STATUS_CHOICES,
+            verbose_name = 'Participation status')
 
     time_factor = models.DecimalField(
             max_digits=10, decimal_places=2,
-            default=1)
+            default=1,
+            help_text="NEED HELP TEXT",
```

Comment:
help_text="Multiplier for time available on time-limited flows"

(NB: Time-limited flows are currently unimplemented.)